app/assets/javascripts/search.js

```diff
@@ -0,0 +1,101 @@
+;(function (global) {
+  'use strict'
+
+  var $ = global.jQuery
+
+  var SearchResultsContainer = (function () {
+    var $searchResultContainer
+    var $results
+
+    var _setSearchDataAttributesOnResults = function () {
+      /*
+        Add the lowercased search terms to a 'data-search' attribute
+        for each possible search result
+       */
+      $results.each(function () {
+        var searchTerms = $(this).find('h2').text()
+        $(this).attr('data-search', searchTerms.toLowerCase())
```

Comment:
Possibly clearer to use the specific [`data`](http://api.jquery.com/data/) API here.